README.md

```diff
@@ -14,18 +14,59 @@ This solves many use cases like:
 
 # Features
 
-  - A side view to display product hierarchy.
+  - A left pane to display project hierarchy.
   - Search functionality for faster access to VIs.
-  - Access to support for multi frame structures like case structures, event structures, disable structures and stacked structures.
+  - Support for viewing Multi Frame Structures like Case Structures, Event Structures, Disable Structures and Stacked Structures.
   - Navigation to SubVIs and Dynamic Dispatch SubVIs.
-  - Context help for SubVIs and most of the function nodes.
+  - Context help for SubVIs and most of the primitive nodes.
+
+Note: FINALE is not supported and this is mostly internal tooling that we are exposing. This is more of a 'hack' that we built, this is not polished code/design.
```

Comment:
> and this is mostly internal tooling that we are exposing. This is more of a 'hack' that we built, this is not polished code/design.

I think this should be rephrased. We could instead say that this is a work in progress, and is not feature complete.

Reply:
Fixed.